package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fas-ui",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
might want to bump this to 1.0.6 to match the github release version

Reply:
I think we don't want to increase the version number like that. when multiple PR comes with versions, it will not keep up with release version

Reply:
Sounds good to me.